app/routes/_gcn.circulars.$circularId/route.tsx

```diff
@@ -7,42 +7,45 @@
  */
 import type { DataFunctionArgs, HeadersFunction } from '@remix-run/node'
 import { json } from '@remix-run/node'
-import { Link, useLoaderData } from '@remix-run/react'
-import { Button, ButtonGroup, Icon } from '@trussworks/react-uswds'
+import { Link, Outlet, useLoaderData } from '@remix-run/react'
+import { ButtonGroup, Icon } from '@trussworks/react-uswds'
 
-import { get } from '../_gcn.circulars/circulars.server'
-import { PlainTextBody } from './Body'
-import { FrontMatter } from './FrontMatter'
-import { origin } from '~/lib/env.server'
+import { getUser } from '../_gcn._auth/user.server'
+import { getVersions, moderatorGroup } from '../_gcn.circulars/circulars.server'
+import { feature, origin } from '~/lib/env.server'
 import { getCanonicalUrlHeaders, pickHeaders } from '~/lib/headers.server'
 import { useSearchString } from '~/lib/utils'
-import type { BreadcrumbHandle } from '~/root/Title'
+import { useFeature } from '~/root'
 
-export const handle: BreadcrumbHandle<typeof loader> = {
-  breadcrumb({ data }) {
-    if (data) {
-      const { circularId, subject } = data
-      return `${circularId}: ${subject}`
-    }
-  },
-}
-
-export async function loader({ params: { circularId } }: DataFunctionArgs) {
+export async function loader({
+  params: { circularId, version },
```

Comment:
`version` is present in a child route, but not in _this_ route. According to https://reactrouter.com/en/main/hooks/use-params, "child routes inherit all params from their parent routes." However, the documentation does _not_ say that parent routes have access to their child routes' params. This would rely on undocumented behavior, which might go away as a future "bug fix."

Also, the loader does not need to return URL params that it doesn't actually use for any computations, as these are available to the route component from the `useParams()` hook.